plugins/opa/examples/fresh-deployment/policy.yaml

```diff
@@ -0,0 +1,24 @@
+output:
+  title: "Deployment is out of date"
+  severity: 0.1
+  remediation: "Update this deployment"
+  category: "Reliability"
+rego: |
+        package fairwinds
+        blockedNamespace(elem) {
+          ns := elem.parameters.blocklist[_]
+          elem.metadata.namespace == ns
+        }
+        latestReplicaDateStale(replicaSets, elem) {
+          rs := replicaSets[_]
+          owner := rs.metadata.ownerReferences[_]
+          owner.uid == elem.metadata.uid
+          rs.status.replicas > 0
+          comparisonDate := time.add_date(time.now_ns(), 0, 0, 0 - input.parameters.days)
+          time.parse_rfc3339_ns(rs.metadata.creationTimestamp) < comparisonDate
+        }
+        hparequired[description] {
+          not blockedNamespace(input)
```

Comment:
I wonder if we should be putting this logic into the configuration rather than the rego - seems like it'd be pretty common